Mu2eG4/inc/Mu2eG4WorkerRunManager.hh

```diff
@@ -49,9 +49,9 @@ namespace mu2e {
     void initializeThread(Mu2eG4MTRunManager* mRM,
                           const G4ThreeVector& origin_in_world);
     void initializeUserActions(const G4ThreeVector& origin_in_world);
-    void initializeRun(art::Event* art_event);
-    void processEvent(art::Event*);
-    G4Event* generateEvt(G4int i_event);
+    void initializeRun(art::Event* const art_event);
+    void processEvent(art::EventID);
+    G4Event* generateEvt(art::EventID);
```

Comment:
Suggest const art::EventID& instead of art::EventID here to avoid making copies, for both functions.